### PR TITLE
Update README with the new design and Lhm adapter

### PR DESCRIPTION
**Pull request by sauloperez**
The README was quite out of date. I removed all the old stuff, explained the new approach and how this affects the usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,17 @@
-# PerconaMigrator [![Build Status](https://travis-ci.org/redbooth/percona_migrator.svg?branch=master)](https://travis-ci.org/redbooth/percona_migrator) [![Code Climate](https://codeclimate.com/github/redbooth/percona_migrator/badges/gpa.svg)](https://codeclimate.com/github/redbooth/percona_migrator)
+# Percona Migrator [![Build Status](https://travis-ci.org/redbooth/percona_migrator.svg?branch=master)](https://travis-ci.org/redbooth/percona_migrator) [![Code Climate](https://codeclimate.com/github/redbooth/percona_migrator/badges/gpa.svg)](https://codeclimate.com/github/redbooth/percona_migrator)
 
-Percona Migrator is a tool for running online and non-blocking
-DDL `ActiveRecord::Migrations` using `pt-online-schema-change` command-line tool of
+Percona Migrator is an **ActiveRecord connection adapter** that allows running
+**MySQL online and non-blocking DDL** `ActiveRecord::Migration`s without needing
```
Comment (ivanilves) [inline README.md]: `ActiveRecord::Migration` without "s" please.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-PERCONA_TOOLKIT var set to ensure all these migrations use Percona in production.
+which runs MySQL alter table statements without downtime.
 
 ## Installation
 
-Percona Migrator relies on `pt-online-schema-change` from  [Percona
+Percona Migrator relies on `pt-online-schema-change` from [Percona
 Toolkit](https://www.percona.com/doc/percona-toolkit/2.0/pt-online-schema-change.html)
 
 For mac, you can install it with homebrew typing `brew install percona-toolkit`. For
```
Comment (ivanilves) [inline README.md]: Mac & Linux please :trollface:

**Comment by ivanilves** [comment]
:speech_balloon: :+1:

File: README.md (hunk 2 above)
Comment (franciscoj) [inline README.md]: At least for Ubuntu, please.

Comment (franciscoj) [inline README.md]: BTW, on arch linux I also needed to install `perl-dbi` and `perl-dbd-mysql`

**Comment by franciscoj** [comment]
:speech_balloon: :+1:

File: README.md (hunk 2 above)
Comment (sauloperez) [inline README.md]: Can you please guys add the installation steps for your platforms? I don't have an easy way to reproduce the steps :trollface:

**Comment by ivanilves** [comment]
I love it... but... Don't have enough power to merge it :grin:

**Pull request merged**